package/MDAnalysis/analysis/accesibility.py

```diff
@@ -0,0 +1,126 @@
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+import numpy as np
+
+from scipy.spatial import KDTree
+from numpy import arange, pi, sin, cos, arccos
+from .base import AnalysisBase
+
+
+ATOMIC_RADI = {
+    "H": 1.200,
+    "HE": 1.400,
+    "C": 1.700,
+    "N": 1.550,
+    "O": 1.520,
+    "F": 1.470,
+    "NA": 2.270,
+    "MG": 1.730,
+    "P": 1.800,
+    "S": 1.800,
+    "CL": 1.750,
+    "K": 2.750,
+    "CA": 2.310,
+    "NI": 1.630,
+    "CU": 1.400,
+    "ZN": 1.390,
+    "SE": 1.900,
+    "BR": 1.850,
+    "CD": 1.580,
+    "I": 1.980,
+    "HG": 1.550,
+}
+
+
+def atomic_radi(e):
+    """van der Waals radii"""
+    DEFAULT_ATOMIC_RADI = 2.0
+    return ATOMIC_RADI[e] if e in ATOMIC_RADI else DEFAULT_ATOMIC_RADI
+
+
+class SASA(AnalysisBase):
+    r"""Calculation of solvent accesible surface areas using Shrake-Rupley
+    'rolling ball' algorithm
+    """
+
+    def __init__(self, ag, n_points=100, probe_radius=1.40, **kwargs):
+        """Parameters
+        ----------
+        ag : AtomGroup
+            Atom group used for the calculation.
+        n_points: int (optional)
+            Number of points used for the estimation of the atom surface area
+        probe_radius: double ( optional )
+            Radius of the probe atom, by default watter radious
+        """
+        super(SASA, self).__init__(ag.universe.trajectory, **kwargs)
+        self._ag = ag
+        self._atom_neighbours = KDTree(ag.positions, 10)
+        self._n_points = n_points
+        self._sphere = self._compute_sphere()
+        self._twice_max_radi = (max(list(ATOMIC_RADI.values())) + probe_radius) * 2
+        self._probe_radius = probe_radius
+
+    def _compute_sphere(self):
+        """Fibonacci lattice to evently distribute points in the sphere."""
+        golden_ratio = (1 + 5**0.5) / 2
+        i = arange(0, self._n_points)
+        theta = 2 * pi * i / golden_ratio
+        phi = arccos(1 - 2 * (i + 0.5) / self._n_points)
+        x_points, y_points, z_points = (
+            cos(theta) * sin(phi),
+            sin(theta) * sin(phi),
+            cos(phi),
+        )
+        return np.transpose(np.array([x_points, y_points, z_points]))
+
+    def _compute(self):
+        asas = np.zeros(len(self._ag.atoms), dtype=np.double)
+
+        for atom in self._ag.atoms:
+            r_w = atomic_radi(atom.type)
+            raddi = r_w + self._probe_radius
```

Comment:
```suggestion
            radii = r_w + self._probe_radius
```
But maybe `radius` (singular) is more pertinent here? See also the other occurrences.